packages/web-config-server/examples.http

```diff
@@ -1,29 +1,39 @@
-@hostname = localhost
-@port = 8000
-@host = {{hostname}}:{{port}}
-@version = v1
+@localhost = http://localhost:8000
+@dev = https://dev-config.tupaia.org
+@host = {{localhost}}
+# Uncomment the line below to point at dev rather than localhost
+# @host = {{dev}}
 @contentType = application/json
 
-# In order to setup authorization, please set 'email' and 'password' in your restClient environement variables 
+# In order to setup authorization, please set 'email' and 'password' in your restClient environement variables
 # see: https://marketplace.visualstudio.com/items?itemName=humao.rest-client#environment-variables
 @authorization  = Basic {{email}}:{{password}}
 
+### Login
 # @name login
-POST http://{{host}}/api/{{version}}/login HTTP/1.1
+POST {{host}}/api/v1/login HTTP/1.1
 content-type: {{contentType}}
 
 {
     "emailAddress": "{{email}}",
     "password": "{{password}}"
 }
 
-###
-
+### Logout
 # @name logout
-POST http://{{host}}/api/{{version}}/logout HTTP/1.1
+GET {{host}}/api/v1/logout HTTP/1.1
```

Comment:
Weirdly the logout endpoint on web-config-server is `GET`, so have updated this to work